src/software/jetson_nano/services/network/network.cpp

```diff
@@ -27,9 +27,10 @@ std::tuple<TbotsProto::PrimitiveSet, TbotsProto::World> NetworkService::poll(
          robot_status.motor_status().front_right().motor_faults_size() > 0 ||
          robot_status.motor_status().back_left().motor_faults_size() > 0 ||
          robot_status.motor_status().back_right().motor_faults_size() > 0) ||
-        (network_ticks / (thunderloop_ticks + 1.0) <=
-         ROBOT_STATUS_TO_THUNDERLOOP_HZ_RATIO))
+        robot_status.power_status().breakbeam_tripped() != last_breakbeam_state_sent ||
+        network_ticks / (thunderloop_ticks + 1.0) <= ROBOT_STATUS_TO_THUNDERLOOP_HZ_RATIO)
     {
+        last_breakbeam_state_sent = robot_status.power_status().breakbeam_tripped();
```

Comment:
I believe all fields in Proto3 are optional, so to be safe, in both of these spots we should check whether there's a `power_status` msg, then check the breakbeam value. Similar to how we do it in Sensor Fusion:

https://github.com/UBC-Thunderbots/Software/blob/38bcbb1a49b375b49750b4c537b79238d41b53f0/src/software/sensor_fusion/sensor_fusion.cpp#L173-L174